flink-table/flink-sql-client/src/test/resources/sql/table.q

```diff
@@ -964,7 +998,7 @@ CREATE TABLE IF NOT EXISTS daily_orders (
  PRIMARY KEY(dt, `user`) NOT ENFORCED
 ) PARTITIONED BY (dt) WITH (
  'connector' = 'filesystem',
- 'path' = '$VAR_BATCH_PATH',
+ 'path' = '/var/folders/xd/9dp1y4vd3h56kjkvdk426l500000gp/T/junit5781726749537647921/7bf278b4-9932-4e24-af7b-1126df4667405030013249695173150',
```

Comment:
this i didn't get...
where does this magic name come from?

Reply:
> this i didn't get... can you please elaborate where does this magic name come from?

Good catch! `$VAR_BATCH_PATH` is defined in `CliClientITCase` and serves as a placeholder to be replaced at runtime. I overwrote this by mistake and will revert this.